server/lib/poller.js

```diff
@@ -65,29 +65,61 @@ class Poller
         }
 
         try {
-
             // Check accounts and operations!
-            let checkAccounts = await Config.findOrCreateDefaultBooleanValue(
+            let checkAccounts = false;
```

Comment:
Wow, that might be a pre-existing error, sorry about that. Let's hoist the declaration out of all the try/catch blocks instead.